### PR TITLE
CSS change to allow easier integration into existing Rails stack

### PR DESCRIPTION
**Pull request by duffyjp**
* Adds CSS id `graphiql-rails` to `html` tag
* Includes the id in the `html` and `body` css rules.

With these changes, I'm able to require your CSS / JS base `application` files without effectively altering my host app's `html` and `body` rules.  I'm doing that so I can embed GraphiQL in my existing stack with authentication, look-n-feel etc.

As I'm using it | When used normally (nothing changes)
---------------|--------------------
Screen Shot 2019-12-20 at 10 05 58 AM | Screen Shot 2019-12-20 at 10 07 00 AM

To do this I copied in two files making changes to use our layout.
* `/controllers/graphiql/rails/editors_controller.rb`
* `/views/graphiql/rails/editors/show.html.erb`

### Host application.css
```css
 *= require graphiql/rails/application
```

### Host application.js
```
//= require graphiql/rails/application
```

# TLDR;
* To avoid requiring versioned CSS files that will cause headaches with graphiql-rails updates, I added a CSS id to increase specificity.

### CODE REVIEW

**Comment by duffyjp** [comment]
I ran into https://github.com/rmosolgo/graphiql-rails/issues/70 when actually using this.  I'm going to close for now since you can't actually use this in production as I expected.
